lib/seqfor.h

```diff
@@ -22,28 +22,26 @@
 namespace qsim {
 
 struct SequentialFor {
-  static uint64_t GetIndex0(
-      uint64_t size, unsigned num_threads, unsigned thread_id) {
+  explicit SequentialFor(unsigned num_threads) {}
+
+  static uint64_t GetIndex0(uint64_t size, unsigned thread_id) {
```

Comment:
Is there a benefit to leaving these methods as static? I could see it causing confusion if a future user was trying to create a "For" implementation and used this as reference.

Reply:
Well, the only benefit is that those functions can be called without creating an object. I guess there is also no benefit to make these methods non-static except for the reason you mentioned. I added a comment.